File: src/services/code_action/code_action_utils.ml
(*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *)

let layout_options options =
  let open Js_layout_generator in
  {
    default_opts with
    bracket_spacing = Options.format_bracket_spacing options;
    single_quotes = Options.format_single_quotes options;
  }

File: rust_port/crates/flow_services_code_action/src/code_action_utils.rs
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

use flow_common::options::Options;
use flow_parser_utils_output::js_layout_generator::Opts;
use flow_parser_utils_output::js_layout_generator::TrailingCommas;

pub fn layout_options(options: &Options) -> Opts {
    Opts {
        bracket_spacing: options.format.bracket_spacing,
        preserve_formatting: false,
        single_quotes: options.format.single_quotes,
        trailing_commas: TrailingCommas::All,
    }
}
